README.md

```diff
@@ -138,5 +138,104 @@ bash.curl.security.curl-pipe-bash.curl-pipe-bash,0.033
 bash.lang.security.ifs-tampering.ifs-tampering,0.044
 ````
 
+## Scala Transformation
 
+Scala is currently not on our list of [supported languages](https://semgrep.dev/docs/supported-languages/#semgrep-supply-chain) for SSC. However, Scala is a Java derivative. And while Java rules can attend the Scala party, reachability rules are like that one friend who didn't get the invite. So for now, Scala's flying solo with parity and lockfile support, no reachability plus-one. That being said, we don’t have any specific support for Scala ecosystems/package files.
 
+A few customers use SBT to construct their dependency tree. For those unfamiliar, a guide to use SBT can be found [here](https://www.baeldung.com/scala/sbt-dependency-tree).
+
+### SBT Dependency Tree Output
+
+Using the **`sbt dependencyTree`** command yields a dependency tree that is SBT-specific but akin to the one required by SSC.
+
+To run this command, the user first needs to add the appropriate plugin:
+**`addDependencyTreePlugin`** 
+
+Or for sbt ≤ 1.3, 
+`addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")` 
+
+(see https://github.com/sbt/sbt-dependency-graph)
+
+If any of your Scala projects have deeply nested dependencies, you probably will also want to run `sbt 'set asciiGraphWidth := 9999'` . This prevents SBT from truncating the dependency information, which will most likely lead to errors in the later transform step if not addressed.
+
+### Transform script
+
+SSC scans require a **`maven_dep_tree.txt`** file in a format which is generated by the command mentioned [here](https://semgrep.dev/docs/semgrep-supply-chain/setup-maven/). To bridge the gap between the SBT dep tree and the Maven format, we approached the transformation using two methods:
+
+1. **Python Script Transformation**
+    
+    A Python script was developed to convert the SBT `dependencyTree` output to a Maven-compatible format. The script can be run from the root of the Scala project (after setting up the `dependencyTree` SBT plugin discussed above) with the following command:
+    
+    `python3 transform.py > maven_dep_tree.txt` 
+    You can find the Python script under the scala folder.
+    
+2. **SBT Command Transformation**
+    
+    An SBT task was written to automatically transform the output and write it to a **`maven_dep_tree.txt`** file, rather than requiring a dependency on an external Python script. 
```

Comment:
```suggestion
    Use an SBT task to automatically transform the output and write it to a `maven_dep_tree.txt` file, rather than requiring a dependency on an external Python script. 
```